bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/handler/HDPowerViewHubHandler.java

```diff
@@ -162,6 +163,7 @@ public void initialize() {
             return;
         }
 
+        pendingShadeInitializations = new ConcurrentHashMap<>();
```

Comment:
If the HashMap is initialized as `final` then it should not be initialized again..

Reply:
That would mean it would have to be `clear()`'ed here. I think it will be faster to create new object and let old one be garbage collected. Haven't tested though, but I don't see any clear advantage of this approach compared to the other?

Reply:
Hmm, reading it again, this is `initialize()`. So it would even have to clear the hash map even upon first initialization. So I really don't see any advantage in having it `final`. It's similar to other instance variables initialized from `initialize()`, like webTargets and futures.

Reply:
Okay, I concur. It's a pickle. Really don't want to make it `@Nullable` just for the sake of this. Clearing an empty hash map should be quite fast anyway. Fixed.